node/src/components/block_proposer/tests.rs

```diff
@@ -17,23 +20,21 @@ use crate::{
 
 const DEFAULT_TEST_GAS_PRICE: u64 = 1;
 
-fn default_gas_payment() -> Gas {
-    Gas::from(1u32)
-}
+static DEFAULT_PAYMENT_AMOUNT: Lazy<Motes> = Lazy::new(|| Motes::new(U512::from(1u64)));
 
 fn generate_transfer(
     rng: &mut TestRng,
     timestamp: Timestamp,
     ttl: TimeDiff,
     dependencies: Vec<DeployHash>,
-    payment_amount: Gas,
+    payment_amount: Motes,
```

Comment:
Is the conversion rate 1 currently hardcoded here anyway? Otherwise it's probably not valid to just change this from `Gas` to `Motes`?

Reply:
So this argument is used to generate runtime args for payment code that accepts tokens (aka Motes or U512). If it's used only for this, I think it's a valid change (considering this argument's meaning), although this is not related to the problem I was solving. I can happily revert changes here if you prefer

Reply:
No, I'm fine with it if the other reviewer(s) are/is; I was just confused.